src/Common/src/TypeSystem/Interop/MethodDesc.Interop.cs

```diff
@@ -28,6 +28,40 @@ public virtual PInvokeMetadata GetPInvokeMethodMetadata()
         {
             return default(PInvokeMetadata);
         }
+
+        /// <summary>
+        /// If <see cref="IsPInvoke"/> is true, retrieves the parameters of the platform invoke.
+        /// </summary>
+        public virtual PInvokeArgumentMetadata[] GetPInvokeArgumentMetadata()
+        {
+            return default(PInvokeArgumentMetadata[]);
+        }
+    }
+
+    [Flags]
+    public enum PInvokeArgumentAttributes
+    {
+        None = 0,
+        In = 1,
+        Out = 2,
+        Lcid = 4,
```

Comment:
What is the `Lcid` and `Retval` value? I don't see it in the ECMA spec or in CLR's `CorParamAttr` enum.